nlohmann-json I think breaks the installation

Update: I found out I can install the package just with:
apt install nlohmann-json3-dev
Would a hint in the readme be helpfull?


### 👓 What did you see?

[nlohmann-json](https://github.com/nlohmann/json) I think breaks the installation

```
CMake Error at src/CMakeLists.txt:3 (find_package):
  By not providing "Findnlohmann_json.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "nlohmann_json", but CMake did not find one.

  Could not find a package configuration file provided by "nlohmann_json"
  (requested version 3.10.5) with any of the following names:

    nlohmann_jsonConfig.cmake
    nlohmann_json-config.cmake

  Add the installation prefix of "nlohmann_json" to CMAKE_PREFIX_PATH or set
  "nlohmann_json_DIR" to a directory containing one of the above files.  If
  "nlohmann_json" provides a separate development package or SDK, be sure it
  has been installed.
```

### ✅ What did you expect to see?

The installation downloads the stuff it needs by itself. Like gtest, ...
In the repository there are about 10 files in the release. Which one do I need? Also there seems to be not really an Installation Instruction. Where to install it best?

### 📦 Which tool/library version are you using?

debian:latest

### 🔬 How could we reproduce it?

Install on debian:latest

### Steps to reproduce the behavior:
As advised in the Readme.md

Tank you

--- a/README.md
+++ b/README.md
@@ -13,8 +13,7 @@ If you need to ask a question, don't open a ticket on GitHub! Please post
 your question on the Cucumber discussion group instead, prefixing the title
 with [CPP].
 
-If you want to contribute code to the project, guidelines are in the
-[`CONTRIBUTING.md` file](https://github.com/cucumber/cucumber-cpp/blob/main/CONTRIBUTING.md).
+If you want to contribute code to the project, guidelines are in [CONTRIBUTING.md](CONTRIBUTING.md).
 
 It relies on a few executables:
 
@@ -30,11 +29,11 @@ It relies on a few libraries:
   Optional for the GTest driver. By default downloaded and built by CMake.
 * [GMock](http://code.google.com/p/googlemock/) 1.6 or later.
   Optional for the internal test suite. By default downloaded and built by CMake.
-* [nlohmann-json](https://github.com/nlohmann/json) 3.10.5 or later
+* [nlohmann-json](https://github.com/nlohmann/json) 3.10.5 or later.
 * [Qt 4 or 5](http://qt-project.org/). Optional for the CalcQt example and QtTest driver (only Qt 5).
 
-It might work with earlier versions of the libraries, but it was not
-tested with them.
+It might work with earlier versions of the libraries, but it was not tested with them.
+See the [CI scripts](.github/workflows/run-all.yml) for details about dependency installation.
 
 Cucumber-Cpp uses the wire protocol at the moment, so you will need
 Cucumber-Ruby installed and available on the path. It is also needed

